x-pack/metricbeat/module/aws/utils.go

```diff
@@ -92,6 +96,52 @@ func GetListMetricsOutput(namespace string, regionName string, period time.Durat
 	return metricWithAccountID, nil
 }
 
+// GetRestAPIsOutput function gets results from apigw api.
+// GetRestApis Apigateway API is used to retrieve only the REST API specified info. This returns a map with the names and ids of RestAPIs configured
+// Limit variable defines maximum number of returned results per page. The default value is 25 and the maximum value is 500.
+func GetRestAPIsOutput(svcRestApi *apigateway.Client, limit *int32) (map[string]string, error) {
+	input := &apigateway.GetRestApisInput{}
+	if limit != nil {
+		input = &apigateway.GetRestApisInput{
+			Limit: limit,
+		}
+	}
+	ctx, cancel := getContextWithTimeout(DefaultApiTimeout)
+	defer cancel()
+	result, err := svcRestApi.GetRestApis(ctx, input)
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving GetRestApis %w", err)
+	}
+
+	// Iterate and display the APIs
+	infoRestAPImap := make(map[string]string, len(result.Items))
+	for _, api := range result.Items {
+		infoRestAPImap[aws.StringValue(api.Name)] = aws.StringValue(api.Id)
+	}
+	return infoRestAPImap, nil
+}
+
+// GetAPIsOutput function gets results from apigatewayv2 api.
+// GetApis Apigateway API is used to retrieve the HTTP and WEBSOCKET specified info. This returns a map with the names and ids of relevant APIs configured
+func GetAPIsOutput(svcHttpApi *apigatewayv2.Client) (map[string]string, error) {
```

Comment:
same here?

Reply:
Replaced to GetAPIGatewayRestAPIOutput and GetAPIGatewayAPIOutput